release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2989,3 +2989,23 @@ link:https://access.redhat.com/solutions/6513261[{product-title} 4.7.37 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-7-38"]
+=== RHBA-2021:4802 - {product-title} 4.7.38 bug fix and security update
+
+Issued: 2021-12-01
+
+{product-title} release 4.7.38 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4802[RHBA-2021:4802] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:4801[RHSA-2021:4801] advisory.
```

Comment:
I get a 404 for both of the advisory links. Is that expected?

Reply:
Links won't work until tomorrow :)